modules/compatibility/src/test/java/org/apache/ignite/compatibility/sql/model/ModelUtil.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.compatibility.sql.model;
+
+import java.util.Random;
+
+/** */
+public final class ModelUtil {
+    /** */
+    private static final String ALPHABETH = "qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM1234567890_";
+
+    /**
+     * Generate random alphabetical string.
+     *
+     * @param rnd Random object.
+     * @param minLen Minimum length of string.
+     * @param maxLen Maximal length of string.
+     * @return Random string object.
+     */
+    public static String randomString(Random rnd, int minLen, int maxLen) {
```

Comment:
Is it copy-pasted from GridTestUtil?

Reply:
Have no idea, it's kinda legacy code from Roman. Also it may be artefact of porting from AI codebase.
It's better to get rig of ModelUtils at all =)

Reply:
Well I've got the answer: the AI has not such method, so if we are wanted to run benchmarks against AI, we have to provide it explicitly